hkmc2DiffTests/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -141,19 +131,9 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
         output(s"Pretty Lowered:")
         output(Printer.mkDocument(le)(using summon[Raise], nestedScp).mkString())
       
-      // NOTE: `blockPreamble` should still take care of those vars that are generated during or after the lowering stage, 
-      // while `Scoped` more reflects the declared vars in the source (or elaborated?) level?
-      // Or, during the lowering, we should also find the correct tmp vars and add them to the set in `Scoped`
-      val varsFromScopedStr =
-        val vars = toplvlDefinedVars.toArray.sortBy(_.uid).map(l => l -> baseScp.allocateName(l))
-        (if vars.isEmpty then doc"" else
-        doc" # let " :: vars.map: (_, nme) =>
-          nme
-        .toList.mkDocument(", ")
-        :: doc";").stripBreaks.mkString(100)
       val (pre, js) = nestedScp.givenIn:
         jsb.worksheet(le)
-      val preStr = pre.stripBreaks.mkString(100) + varsFromScopedStr
+      val preStr = pre.stripBreaks.mkString(100) //+ varsFromScopedStr
```

Comment:
obsolete comment?

Reply:
Please just mark the conversation as resolved yourself when the fix is done and obvious, and no need to comment.